src/features.js

```diff
@@ -31,14 +31,14 @@ export let featureDetectionPromise = Promise.resolve(dynamicImportCheck).then(()
     iframe.style.display = 'none';
     iframe.setAttribute('nonce', nonce);
     function cb ({ data }) {
-      // failed feature detection (security policy) -> revert to default assumptions
-      if (Array.isArray(data)) {
-        supportsImportMaps = data[0];
-        supportsImportMeta = data[1];
-        supportsCssAssertions = data[2];
-        supportsJsonAssertions = data[3];
+      const isFeatureDetectionMessage = Array.isArray(data) && data[0] === 'esms'
```

Comment:
We could potentially make this a bit more resistant to forgery by generating some random bytes instead of this hardcoded string. But I don't see what a potential attacker has to gain by taking over feature detection, so there might not be a need?

Reply:
Agreed, we don't need to be forge resistant.